qd.lisp

```diff
@@ -454,6 +457,26 @@
             (aq+ taken (aq* current-price given))
             (aq+ given (aq* current-price taken)))))))
 
+(defun split-profit-estimates (maker)
+  ;; quite possibly the worst code written in my life, SO FAR ...
+  (multiple-value-bind (rtaken rgiven utaken ugiven)
+      (loop for skip = nil then (cons (car tail) skip)
```

Comment:
refactor idea:

after the full trade log is kept on disk, delete all the "skip" executions from the existing objects, to tighten up offered liquidity